index.css

```diff
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+.header1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: auto;
+  padding: 30px 0;
+}
+
+.header1 h3 {
+  display: flex;
+  flex-direction: row;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  margin-left: 12px;
+  margin-top: 4px;
+}
+
+.header1 h3 a {
+  text-decoration: none;
+}
+
+.nav {
+  padding: 8px;
+}
+
+.ul {
+  list-style: none;
+}
+
+.section {
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/header-shapes mobile.png");
+  background-size: cover;
+  width: 90%;
+  margin: auto;
+  padding: 100px 0;
+}
```

Comment:
- [ ] Good job using flex for the layout :clap: , however please see that currently your design is missing some parts of the background image, this is because you have set the hardcoded 100px padding on the whole section :point_down: 
![image](https://user-images.githubusercontent.com/97521557/212326800-2c293fc9-f46d-4deb-844d-4a446c73fce6.png)

I would suggest to avoid hardcoded values and use flex wherever possible, It gives you more freedom and is totally responsive flex ninja :ninja: , loose the padding, and instead give some gap:20px or something between your flex items it will look good see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/212327135-15465ace-6d42-41dd-a6f6-5ea02e3876e3.png)